example.config.yml

```diff
@@ -50,7 +50,7 @@ drupal_core_path: "/var/www/decoupled.dev/drupal"
 drupal_domain: "{{ vagrant_hostname }}"
 drupal_site_name: "Decoupled Blocks"
 drupal_install_profile: standard
-drupal_enable_modules: [ 'devel' ]
+drupal_enable_modules: [ 'devel', 'pdb', 'pdb_ng2', 'pdb_react', 'admin_toolbar_extra', 'panels', 'page_manager_ui' ]
```

Comment:
dunno if we want pdb_ng2 and pdb_react on by default... maybe just pdb_ng2 for now?